tests/backends/storage/test_swift.py

```diff
@@ -0,0 +1,291 @@
+"""Tests for Ralph swift storage backend"""
+
+import datetime
+import json
+import logging
+
+import pytest
+
+from ralph.backends.storage import swift as swift_module
```

Comment:
I would remove this as it's only used to patch `ralph.utils.now` and import `ralph.utils` instead.